__tests__/__fixtures__/output/python/requests/multipart-data.py

```diff
@@ -2,9 +2,9 @@
 
 url = "https://httpbin.org/anything"
 
-payload = "-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"; filename=\"hello.txt\"\r\nContent-Type: text/plain\r\n\r\nHello World\r\n-----011000010111000001101001--\r\n"
-headers = {"content-type": "multipart/form-data; boundary=---011000010111000001101001"}
+files = {"foo": open("__tests__/__fixtures__/files/hello.txt", "rb")}
+payload = {"bar": "Bonjour le monde"}
 
-response = requests.request("POST", url, data=payload, headers=headers)
+response = requests.request("POST", url, data=payload, files=files)
```

Comment:
This is what a mixed `multipart/form-data` payload of files and form data looks like, and what is returned:

```
> python3 __tests__/__fixtures__/output/python/requests/multipart-data.py
{
  "args": {}, 
  "data": "", 
  "files": {
    "foo": "Hello World\n"
  }, 
  "form": {
    "bar": "Bonjour le monde"
  }, 
  "headers": {
    "Accept": "*/*", 
    "Accept-Encoding": "gzip, deflate", 
    "Content-Length": "256", 
    "Content-Type": "multipart/form-data; boundary=6a25cd61defb2547b1d6d12796df0925", 
    "Host": "httpbin.org", 
    "User-Agent": "python-requests/2.27.1", 
    "X-Amzn-Trace-Id": "REDACTED"
  }, 
  "json": null, 
  "method": "POST", 
  "origin": "REDACTRED", 
  "url": "https://httpbin.org/anything"
}
```